profiling/profile_sort.py

```diff
@@ -28,13 +28,16 @@
 import modmesh
 import numpy as np
 
+from modmesh.profiling import ProfilingResultPrinter
```

Comment:
Why don't you move the script to `modmesh.profiling.__main__` so that we can just type `python3 -m modmesh.profiling` to run it?  See https://yyc.solvcon.net/en/latest/nsd/02numpy/numpy.html#make-a-package .

Reply:
Could you explain what `python3 -m modmesh.profiling` does? I'd like to understand exactly what I need to do.

Reply:
OK. I'll trying to load JSON files that contains result of profiling.

Reply:
Just move this script to `modmesh.profiling.__main__`.